src/3p-frame.js

```diff
@@ -378,6 +383,44 @@ function getCustomBootstrapBaseUrl(
   return `${url}?${internalRuntimeVersion()}`;
 }
 
+/**
+ * Required flags to apply to the sandbox iframe.
+ * @return {Array<string>}
+ */
+export const getRequiredSandboxFlags = () => [
```

Comment:
I think all of these would fit well within the Preact component. Some of these my be granted by default, such as `allow-scripts` and `allow-same-origin`. Others maybe need to be accepted from the props. Possibly similar to `className` prop with `obj-str` and all.

Reply:
Can you describe what you mean? I could modify these to return `Object<string, boolean>` but `objstr` doesn't take merged objects. We could `+` two `objstr` but that also won't overwrite any of the defaults with given sandbox props or vice versa, so we'd potentially have dupes, right?

Reply:
That was just an idea, which may or may not be convenient. Point being that most of our iframe embeds have the same sandbox setup (`alllow-scripts allow-same-origin etc`) and some might need to be optional based on a specific component or component props.